tests/chocolatey-tests/features/Proxy.Tests.ps1

```diff
@@ -170,7 +170,7 @@ Describe "Proxy configuration (<Name>)" -Tag Proxy, ProxySkip -ForEach $TestCase
         $CliArgumentSet = "CliArgumentSetProxy"
 
         if ($ConfigurationsToTest.System) {
-            Set-ItemProperty -Path 'HKCU:/Software/Microsoft/Windows/CurrentVersion/Internet Settings' -name ProxyServer -Value $SystemSet
+            Set-ItemProperty -Path 'HKCU:/Software/Microsoft/Windows/CurrentVersion/Internet Settings' -name ProxyServer -Value "http=none;https=$SystemSet;"

```

Comment:
What if they have `http=$SystemSet;https=none`? We'd end up taking the `none` value if I read the code above right. Is this a valid registry setting to even do, having these `...=none` values in the string? 🤔

Reply:
In the case of someone setting https to an invalid proxy, I would expect their connections to fail. For the intents and purposes of this test, I think I might actually change it to just `https=$SystemSet` so that it tests the scenario you pointed out above where they only have https proxy configured...